flask_classful.py

```diff
@@ -180,8 +180,19 @@ def make_proxy_method(cls, name):
         i = cls()
         view = getattr(i, name)
 
+        # Since the view is a bound instance method, first make it an actual function
+        # So function attributes work correctly
+        def make_func(fn):
+            @functools.wraps(fn)
+            def inner(*args, **kwargs):
+                return fn(*args, **kwargs)
+            return inner
+        view = make_func(view)
+
+        # Now apply the class decorator list in reverse order
+        # to match memeber decorator order
```

Comment:
I meant it's a typo, your PR looks good to me so I've merged with some minor updates. Thank you very much, I'm going to release v0.11.0 for this PR. Great work, @incognick!